core/src/components/tab-bar/tab-bar.ios.scss

```diff
@@ -1,6 +1,7 @@
 @import "./tab-bar";
 @import "../../themes/ionic.globals.ios";
 @import "../tab-button/tab-button.ios.vars";
```

Comment:
External vars should be imported by the component's variable file if it exists, move to `tab-bar.ios.vars`, see toggle: https://github.com/ionic-team/ionic/blob/master/core/src/components/toggle/toggle.ios.vars.scss#L2